.github/workflows/ci.yml

```diff
@@ -23,5 +23,11 @@ jobs:
         java-version: ${{ matrix.java }}
     - name: print Java version
       run: java -version
+    - uses: actions/cache@v1.0.3
+      with:
+        path: ~/.cache/coursier
```

Comment:
I don't have a `~/.cache/coursier` directory on my machine, is it present in Github Actions at least? Maybe `~/.ivy2`? Same for main `guardrail` repo, now that I'm thinking about it

Reply:
https://get-coursier.io/docs/cache

<img width="851" alt="Screen Shot 2019-11-29 at 4 37 01 PM" src="https://user-images.githubusercontent.com/30938/69890898-91f81280-12c6-11ea-9b75-b173a7789794.png">

Reply:
👍